src/web/components/SiteTour/UpdatesTour.tsx

```diff
@@ -0,0 +1,39 @@
+import Joyride, { Actions, CallBackProps } from 'react-joyride';
+
+import { markTourAsSeen, ShouldShowTour, TourSteps } from './tourStorage';
+
+function callback(data: CallBackProps) {
+  const doNotShowAgainActions: Actions[] = ['close', 'skip', 'stop', 'reset'];
+  if (doNotShowAgainActions.includes(data.action)) markTourAsSeen();
+}
+
+export function UpdatesTour() {
+  const showTour = ShouldShowTour();
+  const actionButtonStyle = {
+    backgroundColor: '#cdf200', // --theme-button
```

Comment:
Sadly, joyride doesn't support classes or css - it sets styles directly on elements 😭